.github/workflows/action.yml

```diff
@@ -0,0 +1,61 @@
+name: Robottelo - CI
+
+on:
+  push:
+  pull_request:
+    types: ["opened", "synchronize", "reopened"]
+
+env:
+    PYCURL_SSL_LIBRARY: gnutls
+
+jobs:
+  codechecks:
+    name: 📃 Code Quality
```

Comment:
Looks like the icon are getting detected here

Reply:
I'd prefer to not have emotes in our pipeline names.